build/config/tv/apps-engineering.list

```diff
@@ -22,7 +22,6 @@ apps/gallery
 apps/keyboard
 apps/music
 apps/pdfjs
-apps/system
```

Comment:
Oh, I see you added IAC for smart-system. So I guess that's why you need to remove the original system app. Let's discuss about this after you're back.